src/coreclr/debug/ee/controller.cpp

```diff
@@ -5826,6 +5826,23 @@ static bool IsTailCallThatReturns(const BYTE * ip, ControllerStackInfo* info)
     return retAddr == tailCallAwareRetAddr;
 }
 
+static bool IsStoreTailCallArgs(const BYTE * ip)
+{
+    TraceDestination trace;
+    if (!g_pEEInterface->TraceStub(ip, &trace) || !g_pEEInterface->FollowTrace(&trace))
+    {
+        return false;
+    }
+
+    MethodDesc* pTargetMD =
+        trace.GetTraceType() == TRACE_UNJITTED_METHOD
+        ? trace.GetMethodDesc()
+        : g_pEEInterface->GetNativeCodeMethodDesc(trace.GetAddress());
```

Comment:
Very minor nit: this looks like a copy of the code in `IsTailCallThatReturns`, can we consolidate this into a shared function?

https://github.com/dotnet/runtime/blob/ecefb5e0412bd05841012efe3d7fd914c400343a/src/coreclr/debug/ee/controller.cpp#L5642-L5651